backend/src/main/java/com/bakdata/conquery/models/query/queryplan/aggregators/specific/value/LastValueAggregator.java

```diff
@@ -34,7 +35,7 @@ public LastValueAggregator(Column column) {
 	@Override
 	public void init(Entity entity, QueryExecutionContext context) {
 		selectedEvent = OptionalInt.empty();
-		date = Integer.MIN_VALUE;
+		date = CDateRange.MIN_VALUE;
```

Comment:
Fällt mir gerade auf dass `CDateRange.MIN_VALUE` ja keine `CDateRange` ist. vielleicht kannst du es in `CDateRange.OPEN_LOWER_BOUND` oder so umbenennen

Reply:
ach, du meinst weil `CDateRange.MIN_VALUE = Integer.MIN_VALUE + 1` ist?